### ISSUE TITLE
SignWriting - Left hand vs Right hand edge cases

### ISSUE THREAD

**Issue by AmitMY** [sign/translate#1]
## TL;DR
Hand should have 4 views:
- front
- side
- back
- other side

Currently, the system doesn't support "other side"

## Expected Behavior
Left hand versus right hand isn't the same as top versus bottom of the symbol palette. Consider the graphic below as the right hand can rotate. The last hand has the palm pointing to the side away from the body. It is a right hand that uses a symbol from the bottom of the symbol palette. This phenomena occurs in other situations, but we haven't fully mapped out when and why. I first ran into this problem myself when I tried to automatically colorize signs to have blue right hands and red left hands. It didn't work. Other programmers and researchers have run into this issue as well.

![image](https://user-images.githubusercontent.com/5757359/111022087-53ae3600-83d9-11eb-81c9-6e91108ec4cb.png)

(Demonstrating these shapes is the ASL sign for only-child)

While I can't programmatically tell the difference between a left hand and a right hand, I still wanted to be able to color left and right hands. It's possible, but a manual process.

![image](https://user-images.githubusercontent.com/5757359/111022111-7fc9b700-83d9-11eb-8300-a5d53cf80108.png)

Another example of left versus right. These are all right hands. The left column has the right hand rotate to the side, ending with the finger pointing down. The right column starts the same, points away from the signer and ends with the finger pointing down.

![image](https://user-images.githubusercontent.com/5757359/111022131-9e2fb280-83d9-11eb-8047-1610b84e9b37.png)

(Reported by Steve Slevinski https://www.facebook.com/groups/SuttonSignWriting/permalink/2879117945741468/?comment_id=2879138809072715)

**Comment by AmitMY** [comment]
While indeed, a frame level model is currently used, and it would be cool to support this as well, the goal is to get away from a frame level model, and so this feature is redundant.
